.github/workflows/build-piraeus-server.yaml

```diff
@@ -25,11 +25,11 @@ jobs:
           cd dockerfiles/piraeus-server
           . ./VERSION.env
 
-          DOCKERHUB_TAG_FULL="${DOCKERHUB_IMAGE}:v${LINSTOR_VERSION//'~'/-}"
-          DOCKERHUB_TAG_SHORT="${DOCKERHUB_IMAGE}:v${SHORT_VERSION//'~'/-}"
+          DOCKERHUB_TAG_FULL="${DOCKERHUB_IMAGE}:v$(echo -n $LINSTOR_VERSION | tr '~' -)"
```

Comment:
If you want to be more specific an alternative could be `echo -n $bla | sed -e 's/~rc/-rc/'`. However you feel like, I don't think the more unconditional `tr` will cause troubles